sycl/plugins/unified_runtime/ur/adapters/level_zero/ur_level_zero_kernel.cpp

```diff
@@ -682,9 +682,29 @@ UR_APIEXPORT ur_result_t UR_APICALL urKernelSetArgMemObj(
 
   ur_mem_handle_t_ *UrMem = ur_cast<ur_mem_handle_t_ *>(ArgValue);
 
+  ur_mem_handle_t_::access_mode_t UrAccessMode = ur_mem_handle_t_::read_write;
+  if (Properties) {
+    assert(Properties->stype ==
+           UR_STRUCTURE_TYPE_KERNEL_ARG_MEM_OBJ_PROPERTIES);
+    switch (Properties->memoryAccess) {
+    case 0:
+      break;
```

Comment:
what is being handled in this case?

Reply:
redundant code, removed https://github.com/intel/llvm/pull/9752/commits/11aeaecb731bfb87264c8748f0723693f7d3aec6